lib/main.coffee

```diff
@@ -32,3 +44,7 @@ module.exports =
 
     newColumns = undefined if newColumns.length is 0
     atom.config.set('wrap-guide.columns', newColumns)
+
+  toggle: ->
+    currentState = atom.config.get('wrap-guide.enabled')
```

Comment:
I initially thought about that too, but that still makes it "annoying" to disable permanently for those people who don't want the wrap guide. I'm open to changing the currently implemented behavior if you think that would be better and/or less confusing. It *is* easier to permanently disable now though, since I exposed a config schema for the package.